src/components/AddPatients/AddPatients.jsx

```diff
@@ -1,11 +1,129 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { createPatient } from '../../slices/patients';
 import styles from './AddPatients.module.css';
 
 function AddPatients() {
+	const dispatch = useDispatch();
+
+	const initialPatientState = {
+		birthdate: '',
+		email: '',
+		lastName: '',
+		maritalStatus: '',
+		middleName: '',
+		name: '',
+		phone: '',
+		range: '2',
+	};
+
+	const [patient, setPatient] = useState(initialPatientState);
+
+	const handleInputChange = (event) => {
+		const { name, value } = event.target;
+		setPatient({ ...patient, [name]: value });
+	};
+
+	const savePatient = () => {
+		const { name, middleName, lastName, email,
+			phone, birthdate, maritalStatus, range } = patient;
+
+		dispatch(createPatient(patient))
+			.unwrap()
+			.then(() => {
+				setPatient({
+					birthdate,
+					email,
+					lastName,
+					maritalStatus,
+					middleName,
+					name,
+					phone,
+					range,
+				});
+			});
+	};
+	const form = useRef();
 	return (
 		<div className={styles.contAddPatients}>
 			<div className={styles.AddPatients}>
-				<h1>Hello Welcome Doctor to Add Patients</h1>
+				<div className={styles.boxAddPatients}>
+					<h1>Nuevo Paciente</h1>
+					<div className={styles.gridAddPatients}>
+						<form ref={form} className={styles.formAddPatients}>
+							<div className={styles.dataAddPatients}>
+								<p>Nombre</p>
+								<input
+									className={styles.inputAddPatients}
+									name="name"
+									onChange={handleInputChange}
+									value={patient.name || ''}
+								/>
+							</div>
+							<div className={styles.dataAddPatients}>
+								<p>Segundo apellido</p>
```

Comment:
this is a misunderstandig of "middlename"?
Middlename is more like a Surname or your second name
in my case Gilary Andrea Franco, my middlename is Andrea

Reply:
Yes, it is second name